src/scripts/get-template-library

```diff
@@ -235,30 +235,37 @@ fi
 
 
 # --continuous-integration: check environment
-# NOTE: currently only Travis is supported
+# NOTE: currently only GitHub Actions or Travis are supported
 if [ ${use_ci_work_dir} -eq 1 ]
 then
     [ ${verbose} -eq 1 ] && echo "Configuring continuous integration mode..."
-    if [ -n "${TRAVIS_BUILD_DIR}" ]
+    if [ -n "${GITHUB_WORKSPACE}" ]
+    then
+        ci_work_dir=${GITHUB_WORKSPACE}
+    elif [ -n "${TRAVIS_BUILD_DIR}" ]
     then
         ci_work_dir=${TRAVIS_BUILD_DIR}
     else
-        echo "Not in a Travis build environment: --continuous-integration invalid"
+        echo "Not in a GitHub Actions or Travis build environment: --continuous-integration invalid"
         usage
     fi
-    # $TRAVIS_REPO_SLUG contains the GitHub user/repo (e.g. quattor/template-library-core)
-    pr_repo=$(echo $TRAVIS_REPO_SLUG | sed -e 's%^quattor/%%')
+    # TRAVIS_REPO_SLUG contains the GitHub user/repo (e.g. quattor/template-library-core)
+    pr_repo="${GITHUB_REPOSITORY/#quattor\///}"
```

Comment:
```suggestion
    pr_repo="${GITHUB_REPOSITORY/#quattor\//}"
```
Otherwise this will put a `/` back I think.

Reply:
Thanks for spotting it, `sed` habit!